IDE/Renesas/e2studio/RX65N/RSK/wolfssl_demo/wolfssl_demo.c

```diff
@@ -59,10 +59,10 @@
     static WOLFSSL_CTX* client_ctx;
 #endif /* TLS_CLIENT */
 
-#define TLSSERVER_IP      "192.168.1.14"
+#define TLSSERVER_IP      "192.168.10.6"
 #define TLSSERVER_PORT    11111
-#define YEAR 2023
-#define MON  3
+#define YEAR 2024
```

Comment:
Is this here because the RTC may not be setup for the example? Can you see about extracting the year and month using `__DATE__`?

Reply:
Thanks for the suggestion. Update to extract YEAR and MONTH from `__DATE__` macro.